### ISSUE TITLE
Consider adding `eslint-plugin-jest-formatting`

### ISSUE THREAD

**Issue by manovotny** [manovotny/eslint-config-get-off-my-lawn#145]
It was discovered that the following code does not produce a lint error.

```js
describe('something', () => {
    beforeEach(() => {
        // TODO
    });
    test('should test something', () => {
        expect(true).toBe(false);
    });
});
```

It is desired to have a blank line between the `beforeEach` and `test` blocks.

I first went down the route of debugging [padding-line-between-statements](https://eslint.org/docs/rules/padding-line-between-statements) thinking that I configured `multiline-block-like` incorrectly, but that wasn't it. Turns out, `multiline-expression` causes it to fire an error correctly, but forces us to put blank links after a slew of other things we don't want to care about too. 

That lead to some googling and discovery of https://github.com/jest-community/eslint-plugin-jest/issues/17, which ultimately led to the promise land of https://github.com/dangreenisrael/eslint-plugin-jest-formatting.

We should look to incorporate that here.

**Comment by manovotny** [comment]
This will be fixed in the [v7 release](https://github.com/manovotny/eslint-config-get-off-my-lawn/pull/173).
